### PR TITLE
Convert data from .yml to .json

### LINKED ISSUE #8: This is using yml. It should probably be using JSON. Sorry.

### PR DESCRIPTION
**Pull request by tir38**
"If the dataviz is all JS, then yml is a little silly for storage format."

[Closes #8]

### CODE REVIEW

**Comment by ckdake** [comment]
The JavaScript in index.html will be need to be updated as out fast of this too

File: index.html [hunk 1]
```diff
@@ -1,81 +1,105 @@
...
 
-              if (educationEntry['start_day'] != null) {
-                var startYear = educationEntry['start_day'].getFullYear();
-              }
 
-              var endYear = currentYear;
-              if (educationEntry['end_day'] != null) {
-                  endYear = educationEntry['end_day'].getFullYear();
-              }
+            jQuery.getJSON('http://localhost:8000/data.json', function (result) {
```
Comment (ckdake) [inline index.html]: This bit is not gonna work in prod

Comment (tir38) [inline index.html]: Lol. I was so proud of myself for figuring this out I didn't go back and look over my work =(

**Comment by ckdake** [comment]
Also, you get bonus points if you can do this without jQuery, want to give it a shot?

**Comment by tir38** [comment]
nope =)

**Pull request closed without merging**